src/application/wizard/service.ts

```diff
@@ -99,7 +101,7 @@ export class WizardService {
             stream.on('end', () => resolve(Buffer.concat(chunks)));
```

Comment:
Upload progress should include this buffer stage too. Not sure what the ratio of buffer to s3 upload should be but maybe 50/50?

Reply:
This is being read from a os.tmpDir. Not sure what value this would add.

Reply:
lets keep this discussion to this comment thread as they're the same thing https://github.com/libero/reviewer-submission/pull/123/files#r393710484

Reply:
By this point, the server already has committed the file to the tmp directory on the server. This is opening a stream to local file.